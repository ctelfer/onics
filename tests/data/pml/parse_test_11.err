parse error on file *END OF INPUT* line 0: 'x' is a scalar global and can not be accessed as a byte string

