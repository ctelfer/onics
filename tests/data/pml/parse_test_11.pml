# Scalar variable index test
#
# This should fail:  can't index a scalar global.
#
int x = 0;

?- x[1,1] == 0 -? { print "equal"; }
