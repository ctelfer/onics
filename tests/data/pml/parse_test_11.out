#########
Initializing parse data structures
#########
#########
Starting Parse
#########
